sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2927,6 +2927,17 @@ object SQLConf {
       // show full stacktrace in tests but hide in production by default.
       .createWithDefault(Utils.isTesting)
 
+  val PYTHON_UDF_PROFILER =
+    buildConf("spark.sql.pyspark.udf.profiler")
+      .doc("Configure the Python/Pandas UDF profiler by enabling or disabling it " +
+        "with the option to choose between \"perf\" and \"memory\" types, " +
+        "or unsetting the config disables the profiler. This is disabled by default.")
```

Comment:
I'm wondering if "setting the config to ... disables the profiler" is more clear than "unsetting the config".

Reply:
I'm not sure .. What do you think the `...` should be in that case?

Reply:
I'm not sure what `...` should be, I guess that is the reason why I suggested not using "unsetting". How do we expect users to "unset" a config, is there a specific API?

Reply:
Sorry I was looking at pyspark.SparkConfig, thanks!